### PR TITLE
logservice: added config files for 3 nodes log service

### PR DESCRIPTION
**Pull request by lni**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [x] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #

**What this PR does / why we need it:**

added config files for 3 nodes log service

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

File: etc/bootstrap-example/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Assuming you are in MO's source directory.
+
+Build mo-service -
+```
+make service
+```
+Start 3 mo-service instances configured as Log Service instances in 3 different terminals on the same machine. 
+
+```
+./mo-service -cfg etc/log-node-1.toml
```
Comment (sukki37) [inline etc/bootstrap-example/README.md:12]:
```
./mo-service -cfg etc/bootstrap-example/log-node-1.toml
./mo-service -cfg etc/bootstrap-example/log-node-2.toml
./mo-service -cfg etc/bootstrap-example/log-node-3.toml
```

**Review by sukki37 — CHANGES_REQUESTED** [review]
the configuration file path for starting `mo-service` in the bootstrap example is incorrect.

File: etc/bootstrap-example/README.md (hunk 1 above)
Comment (wanglei4687) [reply etc/bootstrap-example/README.md:12]: It may like this: ` ./mo-server ./etc/bootstrap-example/log-node-1.toml`

Comment (lni) [reply etc/bootstrap-example/README.md]: fixed

**Review by sukki37 — APPROVED** [review]

**Pull request merged**